src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -898,7 +898,7 @@ EnsureSupportedSequenceColumnType(Oid sequenceOid)
 	{
 		ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
 						errmsg("cannot create an mx table with columns which use "
-							   "sequences, but are not BIGINT")));
+							   "sequences, but are not bigserial")));
```

Comment:
what about changing it to `cannot create an mx table with columns which use sequences, but are not bigserial`. I mean does `columns which use sequences` sound right? Can we come up with a better wording?